OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -190,17 +190,17 @@ public Actor ClosestProc(Actor self, Actor ignore)
 					Location = r.Actor.Location + r.Trait.DeliveryOffset,
 					Actor = r.Actor,
 					Occupancy = self.World.ActorsHavingTrait<Harvester>(h => h.LinkedProc == r.Actor).Count()
-				}).ToLookup(r => r.Location);
+				}).ToDictionary(r => r.Location);
```

Comment:
```suggestion
				}).Where(r => r.Occupancy < Info.MaxUnloadQueue).ToDictionary(r => r.Location);
```

and an !`refineries`.Any() check below

Reply:
Updated